nixos/modules/services/networking/ssh/sshd.nix

```diff
@@ -583,7 +583,7 @@ in
           (lport: "sshd -G -T -C lport=${toString lport} -f ${sshconf} > /dev/null")
```

Comment:
Just to be sure/document the reasoning: 
We don't need the escaping here, as `lport` is always just an integer and thus won't cause any harm.